rust/operator/src/container.rs

```diff
@@ -317,9 +329,13 @@ impl ContainerConfig {
     /// - Namenode ZooKeeper fail over controller (ZKFC)
     /// - Datanode main process
     /// - Journalnode main process
+
+    #[allow(clippy::too_many_arguments)]
     fn main_container(
         &self,
         hdfs: &HdfsCluster,
+        hdfs_name: &str,
+        hdfs_namespace: &str,
```

Comment:
Do we really want to add extra parameters for something i can get from the `HdfsCluster`? 
Same for other methods.
Id prefer to just impl this on the cluster object and call it?

Reply:
Created https://github.com/stackabletech/hdfs-operator/pull/451 based on this review comment, please have a look there